git_machete/cmd.py

```diff
@@ -149,7 +151,16 @@ def __init__(self) -> None:
         self.opt_yes: bool = False
 
 
-def ask_if(cli_ctxt: CommandLineContext, msg: str, opt_yes_msg: Optional[str], apply_fmt: bool = True) -> str:
+def ask_if(
+    cli_ctxt: CommandLineContext,
+    msg: str,
+    opt_yes_msg: Optional[str],
+    apply_fmt: bool = True,
```

Comment:
Nit: I'd swap `apply_fmt` and `override` since `override` is more similar in nature to `opt_yes_msg`